doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -448,6 +448,11 @@ Other libraries
     The library is no longer directly referenced from the Application Event Manager.
     Instead, it uses the Application Event Manager hooks to connect with the manager.
 
+* :ref:`lib_location`:
+
+  * Added sanity check to double check that the currently selected LTE system mode supports GNSS when using the GNSS location method for a location request.
```

Comment:
```suggestion
  * Added sanity check to verify that the currently selected LTE system mode supports GNSS when using the GNSS location method for a location request.
```

Reply:
Nitpicking: I would replace "LTE" with "modem" because LTE spec does certainly not say anything about GNSS.